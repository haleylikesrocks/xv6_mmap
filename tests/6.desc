File Backed mmap test : testing msync
