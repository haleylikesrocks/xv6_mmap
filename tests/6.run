cd src; ./../tester/run-xv6-command.exp CPUS=1 Makefile.test test_6 | grep XV6_TEST_OUTPUT; cd ..
