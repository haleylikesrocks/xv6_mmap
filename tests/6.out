XV6_TEST_OUTPUT : file content now : < Hello World.! >
XV6_TEST_OUTPUT : file open suceeded
XV6_TEST_OUTPUT : mmap suceeded
XV6_TEST_OUTPUT : Before mysnc, content in mmap-ed region: Hello World.!
XV6_TEST_OUTPUT : msync return val : 0
XV6_TEST_OUTPUT : After mysnc, content in the mmap-ed region : This is overwritten content.!
XV6_TEST_OUTPUT : file content now : < This is overwritten content.! >
XV6_TEST_OUTPUT : munmap suceeded
XV6_TEST_OUTPUT : file close suceeded
